plugins/scaffolder/src/components/fields/RepoUrlPicker/BitbucketRepoPicker.tsx

```diff
@@ -18,15 +18,20 @@ import FormControl from '@material-ui/core/FormControl';
 import FormHelperText from '@material-ui/core/FormHelperText';
 import Input from '@material-ui/core/Input';
 import InputLabel from '@material-ui/core/InputLabel';
+import { Select, SelectItem } from '@backstage/core-components';
 import { RepoUrlPickerState } from './types';
 
 export const BitbucketRepoPicker = (props: {
+  allowedOwners?: string[];
   onChange: (state: RepoUrlPickerState) => void;
   state: RepoUrlPickerState;
   rawErrors: string[];
 }) => {
-  const { onChange, rawErrors, state } = props;
+  const { allowedOwners = [], onChange, rawErrors, state } = props;
   const { host, workspace, project } = state;
+  const ownerItems: SelectItem[] = allowedOwners
+    ? allowedOwners?.map(i => ({ label: i, value: i }))
+    : [];
   return (
     <>
       {host === 'bitbucket.org' && (
```

Comment:
This line is a headache for me since we have Bitbucket Server on premise (custom hostname)...
Can we just have a `host.indexOf('bitbucket') >= 0` or similar?

Reply:
Projects (like Workspaces?) or Users can have repositories at Bitbucket Server. Our use case:
https://developer.atlassian.com/server/bitbucket/rest/v804/api-group-project/#api-api-latest-projects-projectkey-repos-post
`projectKey` is the chosen owner (whether Project or User).

Reply:
@vitokodev workspace is **not** project. Bitbucket Cloud has workspace **and** project

Reply:
Got it. That `if` it is for a different purpose. So... since I still have `allowedOwners.length > 0` I should somehow display a `select` on the line 75.

Thanks Patrick, I will figure out a way to solve this specific use case.